set some variables to keep the build system from doing stupid stuff

--- a/lib/Module/Depends/Intrusive.pm
+++ b/lib/Module/Depends/Intrusive.pm
@@ -3,6 +3,7 @@
 use base qw( Module::Depends );
 use Cwd qw( getcwd );
 use ExtUtils::MakeMaker ();
+use Env::Sanctify;
 
 sub _find_modules {
     my $self = shift;
@@ -92,6 +93,7 @@
         no strict;
         no warnings;
 	local $0 = $file;
+        my $sanctify = Env::Sanctify->sanctify( env => { PERL_MM_USE_DEFAULT => 1, PERL_AUTOINSTALL => "--skipdeps" } );
         do "$file";
     };
     $self->error( $@ ) if $@;
